src/src/settings.py

```diff
@@ -81,8 +81,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
```

Comment:
I think sqlite3 will work for your project.

Reply:
Sqlite is back again on the database, I agree with you.